app/controllers/people_controller.rb

```diff
@@ -68,7 +71,7 @@ def inactive
   end
 
   def index
-    @people = Person.active.select{|person| person.department&.name == 'CERT' || person.department&.name == 'Police'}
+    @people = Person.active.select{|person| person.department&.shortname == 'CERT' || person.department&.shortname == 'BAUX'}
```

Comment:
I believe currently this loads all active people, doing the filtering in ruby code instead of at the database level. There may also be an n+1 query problem. See the queries produced for 3 matching people:

```
Person.active.select{|person| person.department&.shortname == 'CERT' || person.department&.shortname == 'BAUX'}
  Person Load (0.6ms)  SELECT "people".* FROM "people" WHERE "people"."status" = $1  ORDER BY division1, division2, title_order, start_date ASC  [["status", "Active"]]
  Department Load (0.3ms)  SELECT  "departments".* FROM "departments" WHERE "departments"."id" = $1 LIMIT 1  [["id", 4]]
  Department Load (0.3ms)  SELECT  "departments".* FROM "departments" WHERE "departments"."id" = $1 LIMIT 1  [["id", 4]]
  Department Load (0.3ms)  SELECT  "departments".* FROM "departments" WHERE "departments"."id" = $1 LIMIT 1  [["id", 4]]
  Department Load (0.3ms)  SELECT  "departments".* FROM "departments" WHERE "departments"."id" = $1 LIMIT 1  [["id", 5]]
```

would this be better ?:

`Person.active.joins(:department).where("departments.shortname": ['CERT', 'BAUX'])`

or to avoid the join entirely by using two queries:

`Person.active.where(department_id: Department.where(shortname: ['CERT', 'BAUX']).pluck(:id))`